src/resources/swagger.yaml

```diff
@@ -1415,6 +1437,13 @@ components:
               value:
                 msg: 'string'
                 code: 'INVALID_FILE_NAME'
+    MissingEntityCreate:
+      description: 'successful upload'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/MissingEntityCreateResp'
```

Comment:
I don't think this needs to be defined outside of where it is used. Can simplify the swagger file a bit by just moving this in place under the response 200 at line 861.